cuda/matrix/dense_kernels.cu

```diff
@@ -117,6 +118,49 @@ void apply(std::shared_ptr<const CudaExecutor> exec,
 GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(GKO_DECLARE_DENSE_APPLY_KERNEL);
 
 
+template <typename ValueType>
+void mem_size_bccoo(std::shared_ptr<const ReferenceExecutor> exec,
+                    const matrix::Dense<ValueType> *source,
+                    const size_type block_size,
+                    size_type *result) GKO_NOT_IMPLEMENTED;
+/*
+{
+
+}
+*/
```

Comment:
Same thing for all 3 functions you added in here.

Reply:
Done